README.md

```diff
@@ -88,17 +88,19 @@ We want to develop an API that learns from the
 
 ### Proposed API
 
-The basic API would be two calls: `.toString(value)` to produce a string,
-and `.toFragment(value)` to produce a DocumentFragment. Sanitizers can
+The basic API would be two calls: `.sanitizeToString(value)` to produce a string,
+and `.sanitize(value)` to produce a DocumentFragment. Sanitizers can
 be constructed with a dictionary of options.
 
 ```
-interface Sanitizer {
-  constructor(SanitizerConfig? config);
-  DOMString toString(DOMString input);
-  DocumentFragment toFragment(DOMString input);
-
-  readonly SanitizerConfig creation_options;
+[
+  Exposed=Window
+] interface Sanitizer {
+  [RaisesException] constructor(SanitizerConfig? config);
```

Comment:
Please remove [RaisesException].

RaisesException is a Blink-specific attribute, telling our code generator to add ExceptionState-references to the callback signature. This doesn't seem to have an equivalent in the standards world.

(Reference: https://chromium.googlesource.com/chromium/src/+/master/third_party/blink/renderer/bindings/IDLExtendedAttributes.md#RaisesException_i_m_a)

Reply:
I recall @yuki3 suggesting we make this optional (e.g. `optional SanitizerConfig config = {}`) rather than nullable (via `?`).

Reply:
You are right and done.